release-notes/swan-lake-beta1-release-note.md

```diff
@@ -45,6 +45,31 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+- Worker deadlock detection has been improved to include the `wait` action.
+- The `self` variable of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is of a type that is not a subtype of `readonly` or `isolated object {}`.
+- The `lang.xml:strip()` function does not mutate an XML value. Therefore, using the `lang.xml:strip()` function on immutable XML values is allowed.
+
+#### Breaking Changes
+
+- An included record parameter of a function can only be specified after any required and/or defaultable parameters of the function.
+- Additive expressions and multiplicative expressions are no longer supported with numeric values when the static types of the operands belong to different numeric basic types.
+- Configurable variables are implicitly `final` now. Moreover, the type of such a variable is now effectively the intersection of the specified type and `readonly`. Therefore, configurable variables no longer support the `final` and `isolated` qualifiers.
+- Type narrowing will no longer take place for captured variables of an anonymous function since the narrowed type cannot be guaranteed during the execution of the function.
+- Type narrowing will now be reset after a compound assignment.
+- Worker message passing after waiting for the same worker has been disallowed.
+- When a named worker is used in a `wait` action, it can no longer be used in a variable reference anywhere else.
+- When the `type-descriptor` is ambiguous, it is considered according to the following table, in which the type precedence is presented in the decreasing order.
```

Comment:
type-descriptor is not a keyword.

```suggestion
- When the type-descriptor is ambiguous, it is parsed according to the following table, in which the precedence is specified in decreasing order.
```

@lochana-chathura it's OK to say parsed here, right?

Reply:
+1